### PR TITLE
docs/user/aws/install_upi: Fix "machinesets" -> "machineset" typo

### PR DESCRIPTION
**Pull request by jdoss**
Creating manifests:

```
$ openshift-install create manifests
WARNING There are no compute nodes specified. The cluster will not fully initialize without compute nodes. 
INFO Consuming "Install Config" from target directory 
```

Produces the following yaml files:

```
$ ls -lah openshift/
total 68K
drwxr-xr-x. 2 jdoss jdoss 4.0K Oct 10 14:39 .
drwxrwxr-x. 4 jdoss jdoss 4.0K Oct 10 14:39 ..
-rw-r--r--. 1 jdoss jdoss  293 Oct 10 14:39 99_binding-discovery.yaml
-rw-r--r--. 1 jdoss jdoss  219 Oct 10 14:39 99_cloud-creds-secret.yaml
-rw-r--r--. 1 jdoss jdoss  181 Oct 10 14:39 99_kubeadmin-password-secret.yaml
-rw-r--r--. 1 jdoss jdoss 1.5K Oct 10 14:39 99_openshift-cluster-api_master-machines-0.yaml
-rw-r--r--. 1 jdoss jdoss 1.5K Oct 10 14:39 99_openshift-cluster-api_master-machines-1.yaml
-rw-r--r--. 1 jdoss jdoss 1.5K Oct 10 14:39 99_openshift-cluster-api_master-machines-2.yaml
-rw-r--r--. 1 jdoss jdoss 2.7K Oct 10 14:39 99_openshift-cluster-api_master-user-data-secret.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-0.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-1.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-2.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-3.yaml
-rw-r--r--. 1 jdoss jdoss 2.7K Oct 10 14:39 99_openshift-cluster-api_worker-user-data-secret.yaml
-rw-r--r--. 1 jdoss jdoss 1.2K Oct 10 14:39 99_openshift-machineconfig_master.yaml
-rw-r--r--. 1 jdoss jdoss 1.2K Oct 10 14:39 99_openshift-machineconfig_worker.yaml
-rw-r--r--. 1 jdoss jdoss  222 Oct 10 14:39 99_role-cloud-creds-secret-reader.yaml
```

Following the documentation it says to remove the following things:

`$ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml openshift/99_openshift-cluster-api_worker-machinesets-*.yaml`

Doing so left the `openshift/99_openshift-cluster-api_worker-machineset-*.yaml` files:

```
$ ls -lah openshift/
total 56K
drwxr-xr-x. 2 jdoss jdoss 4.0K Oct 10 14:44 .
drwxrwxr-x. 4 jdoss jdoss 4.0K Oct 10 14:39 ..
-rw-r--r--. 1 jdoss jdoss  293 Oct 10 14:39 99_binding-discovery.yaml
-rw-r--r--. 1 jdoss jdoss  219 Oct 10 14:39 99_cloud-creds-secret.yaml
-rw-r--r--. 1 jdoss jdoss  181 Oct 10 14:39 99_kubeadmin-password-secret.yaml
-rw-r--r--. 1 jdoss jdoss 2.7K Oct 10 14:39 99_openshift-cluster-api_master-user-data-secret.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-0.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-1.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-2.yaml
-rw-r--r--. 1 jdoss jdoss 2.1K Oct 10 14:39 99_openshift-cluster-api_worker-machineset-3.yaml
-rw-r--r--. 1 jdoss jdoss 2.7K Oct 10 14:39 99_openshift-cluster-api_worker-user-data-secret.yaml
-rw-r--r--. 1 jdoss jdoss 1.2K Oct 10 14:39 99_openshift-machineconfig_master.yaml
-rw-r--r--. 1 jdoss jdoss 1.2K Oct 10 14:39 99_openshift-machineconfig_worker.yaml
-rw-r--r--. 1 jdoss jdoss  222 Oct 10 14:39 99_role-cloud-creds-secret-reader.yaml
```

Removing the `s` from the `rm` command below:

`$ rm -rf openshift/99_openshift-cluster-api_worker-machineset-*.yaml`

Gives us the results we expect!

```
$ ls -lah openshift/
total 40K
drwxr-xr-x. 2 jdoss jdoss 4.0K Oct 10 14:45 .
drwxrwxr-x. 4 jdoss jdoss 4.0K Oct 10 14:39 ..
-rw-r--r--. 1 jdoss jdoss  293 Oct 10 14:39 99_binding-discovery.yaml
-rw-r--r--. 1 jdoss jdoss  219 Oct 10 14:39 99_cloud-creds-secret.yaml
-rw-r--r--. 1 jdoss jdoss  181 Oct 10 14:39 99_kubeadmin-password-secret.yaml
-rw-r--r--. 1 jdoss jdoss 2.7K Oct 10 14:39 99_openshift-cluster-api_master-user-data-secret.yaml
-rw-r--r--. 1 jdoss jdoss 2.7K Oct 10 14:39 99_openshift-cluster-api_worker-user-data-secret.yaml
-rw-r--r--. 1 jdoss jdoss 1.2K Oct 10 14:39 99_openshift-machineconfig_master.yaml
-rw-r--r--. 1 jdoss jdoss 1.2K Oct 10 14:39 99_openshift-machineconfig_worker.yaml
-rw-r--r--. 1 jdoss jdoss  222 Oct 10 14:39 99_role-cloud-creds-secret-reader.yaml
```

Here is the `openshift-install` version that I used.

```
$ openshift-install version
openshift-install v4.1.18-201909201915-dirty
built from commit 80c0ef5e57812daf721522db78972aa557730fc4
release image quay.io/openshift-release-dev/ocp-release@sha256:420633acf3fc7572372fe2df758152f6ab1f53a21c79a6c4b741fa0394c7df3a
```

### CODE REVIEW

**Comment by wking** [comment]
Good catch :).  Can you force-push a fresh commit where you use the more-specific PR subject I just pushed as the commit subject and your very nice PR body message as the commit body?  That makes it easier for folks trying to understand the code with `git blame ...` in the future.

**Comment by wking** [comment]
Looks like [the official docs are right][1].

[1]: https://github.com/openshift/openshift-docs/blame/b02f3fa8a1a3495cdcaf669ec38688ca5b41f011/modules/installation-generate-aws-user-infra-ignition.adoc#L57

**Comment by jdoss** [comment]
@wking Ok done. :smiley:

**Comment by wking** [comment]
/hold

GitHub says you need a rebase.

**Comment by jdoss** [comment]
Ok I fetched upstream on my fork and rebased. Let me know if you need anything else.

**Comment by wking** [comment]
Looks like you merged with eef53c9335cb06f100c60a5e63ded285b083ca30.  Can you rebase instead (so we have one commit here instead of 13)?

File: docs/user/aws/install_upi.md [hunk 1]
```diff
@@ -236,6 +236,7 @@ CSRs can be approved by name, for example:
 oc adm certificate approve csr-bfd72
 ```
 
+
```
Comment (jstuever) [inline docs/user/aws/install_upi.md]: Remove this extra line.

**Comment by jdoss** [comment]
@jstuever Done.

**Pull request merged**